src/SumoCoders/FrameworkCoreBundle/Resources/assets/sass/components/_navs.scss

```diff
@@ -5,3 +5,7 @@
     color: $white;
   }
 }
+
+.nav-tabs {
+  margin-bottom: 20px;
```

Comment:
Shouldn't this be a variable? Something based on `$padding-base-vertical` for instance?